views/js/qtiCreator/widgets/interactions/uploadInteraction/states/Question.js

```diff
@@ -39,25 +41,35 @@ define([
         // Pre-select a value in the types combo box if needed.
         let preselected = uploadHelper.getExpectedTypes(interaction);
 
+        types.unshift({ mime: ANY_KIND, label: __('-- Any kind of file --') });
```

Comment:
**quibble:** Not a blocker, but I wonder if the decorator should be part of the translation. It can be challenged as it has pros and cons...

Reply:
I think translation needed, because we have ability translate labels for mime types here https://github.com/oat-sa/tao-core-sdk-fe/blob/master/src/core/mimetype/definitions.js

Reply:
Well my comment was not about the need to translate but if we should include the decorator in it. In other words, should we do `__('-- Any kind of file --')` or `-- __('Any kind of file') --`